### ISSUE TITLE
Losing color vibrance when using Automatic CFG.

### ISSUE THREAD

**Issue by CocoPuffLeo** [Extraltodeus/ComfyUI-AutomaticCFG#38]
I am currently using pony model with a  cfg of 7, when using without the automatic cfg I was able to get vibrance colors. But when ever I use the sutomatic CFG node the result always looks washed off and less vibrant and look rather grey ish. Is there a method or settings I am missing that could correct this issue?

**Comment by CocoPuffLeo** [comment]
Hmm that is strange, because after I have experienced this color issue I have adjusted the CFG number to 8, 12, 14, even 26 just to see if it will change the result. But even increasing the CFG to extreme numbers it still maintained it's result of less vibrant coloring, and overall details in the image. The majority of the image looks perfect, but the minor details like skin folds, hair strings, skin shadows, and background details are lost. Could this be effected by me using AYS (Align Your Steps, provided by NVIDIA) as the scheduler?

**Comment by Extraltodeus** [comment]
no and it's kinda weird that you can push cfg to 26 and not see any change. Like, what's your workflow?

**Comment by CocoPuffLeo** [comment]
Yeah really weird, here these are the examples I have tested, and you can see the one I use Automatic CFG will look less colorful.
![ComfyUI_01252_](https://github.com/Extraltodeus/ComfyUI-AutomaticCFG/assets/120401853/af04f2da-fbdf-468b-97f6-1be779ace248)
Using Automatic CFG 8
![ComfyUI_01253_](https://github.com/Extraltodeus/ComfyUI-AutomaticCFG/assets/120401853/23c483aa-5d51-4d08-b575-a4ac921b8a7c)
Without using Automatic CFG, normal CFG 8
![ComfyUI_01255_](https://github.com/Extraltodeus/ComfyUI-AutomaticCFG/assets/120401853/a9ab665c-fe90-4460-8e81-4b142e8d7f26)
Using Automatic CFG 12

Workflow
[Testing.json](https://github.com/user-attachments/files/15742460/Testing.json)
